GovUk.Frontend.Umbraco.ExampleApp/uSync/v9/ContentTypes/govukfileuploadsettings.config

```diff
@@ -1,97 +1,81 @@
-	<Info>
-		<Name>File upload settings</Name>
-		<Icon>icon-cloud-upload color-black</Icon>
-		<Thumbnail>folder.png</Thumbnail>
-		<Description>Settings for a file upload component. Ask users to upload something only if it’s critical to the delivery of your service.</Description>
-		<AllowAtRoot>False</AllowAtRoot>
-		<IsListView>False</IsListView>
-		<Variations>Nothing</Variations>
-		<IsElement>true</IsElement>
-		<HistoryCleanup>
-			<PreventCleanup>False</PreventCleanup>
-			<KeepAllVersionsNewerThanDays></KeepAllVersionsNewerThanDays>
-			<KeepLatestVersionPerDayForDays></KeepLatestVersionPerDayForDays>
-		</HistoryCleanup>
-		<Folder>GOV.UK</Folder>
-		<Compositions>
-			<Composition Key="bf69e8d6-97a6-4b50-bec7-c5993e120339">govukCssClasses</Composition>
-			<Composition Key="37cc75ea-be5c-4107-b9b8-0ef34f4b7c5e">govukErrorMessagePrefix</Composition>
-			<Composition Key="d508995b-a958-4273-8ab3-50735220a1b2">govukGrid</Composition>
-			<Composition Key="d7016563-0b6f-4c3a-b75d-b0bbe66419fd">govukGridColumnClasses</Composition>
-			<Composition Key="2bc3c434-f3ff-495d-b360-1a005245602e">govukLabelIsPageHeading</Composition>
-			<Composition Key="23e1a03d-dc42-4318-9e17-d458dde2dba6">govukModelProperty</Composition>
-			<Composition Key="f2365c7e-08c3-4ef9-abbc-e337e34de272">govukValidationRequired</Composition>
-		</Compositions>
-		<DefaultTemplate></DefaultTemplate>
-		<AllowedTemplates />
-	</Info>
-	<Structure />
-	<GenericProperties>
-		<GenericProperty>
-			<Key>cac445e1-9d87-452e-94de-a70e43ff09ff</Key>
-			<Name>Allowed file types</Name>
-			<Alias>errorMessageAllowedFileExtensions</Alias>
-			<Definition>0cc0eba1-9960-42c9-bf9b-60e150b429ae</Definition>
-			<Type>Umbraco.TextBox</Type>
-			<Mandatory>false</Mandatory>
-			<Validation></Validation>
-			<Description><![CDATA[Sets the message displayed if the file uploaded is not one of the supported types/extensions]]></Description>
-			<SortOrder>1</SortOrder>
-			<Tab Alias="errorMessages">Error messages</Tab>
-			<Variations>Nothing</Variations>
-			<MandatoryMessage></MandatoryMessage>
-			<ValidationRegExpMessage></ValidationRegExpMessage>
-			<LabelOnTop>false</LabelOnTop>
-		</GenericProperty>
-		<GenericProperty>
-			<Key>b5083899-5c0c-440f-8295-4983bdeaaa5d</Key>
-			<Name>Maximum file size</Name>
-			<Alias>errorMessageMaxFileSize</Alias>
-			<Definition>0cc0eba1-9960-42c9-bf9b-60e150b429ae</Definition>
-			<Type>Umbraco.TextBox</Type>
-			<Mandatory>false</Mandatory>
-			<Validation></Validation>
-			<Description><![CDATA[Sets the message displayed if the file uploaded exceeds the maximum size]]></Description>
-			<SortOrder>0</SortOrder>
-			<Tab Alias="errorMessages">Error messages</Tab>
-			<Variations>Nothing</Variations>
-			<MandatoryMessage></MandatoryMessage>
-			<ValidationRegExpMessage></ValidationRegExpMessage>
-			<LabelOnTop>false</LabelOnTop>
-		</GenericProperty>
-		<GenericProperty>
```

Comment:
Added MaxFileSize back. It seems to remove it by itself somehow because it was removed after I clicked Save serveral times now.
![image](https://github.com/user-attachments/assets/b1972b13-3c08-47d9-9f51-c93db7de540f)

Reply:
It's not there. Are you updating the copies in both `GovUk.Frontend.Umbraco` and `GovUk.Frontend.Umbraco.ExampleApp`? If you only update the example app, the build will copy the old version back from `GovUk.Frontend.Umbraco`. You need to copy your changes there.

Reply:
Added it back. It's done in this commit: [Add MaxFileSize back to FileUploadSettings](https://github.com/thepensionsregulator/govuk-frontend-aspnetcore-extensions/pull/314/commits/19775c7dfdebe0fd9c069075ca1884220b8cc8ee)